shared/js/background/atb.js

```diff
@@ -175,6 +182,11 @@ const ATB = (() => {
                         return !!atb
                     })
 
+                    // in case there is no assigned atb variant, enroll into Chrome Counteract experiment
+                    if (!atb && settings.getSetting(atb) && settings.getSetting(atb).atb && utils.getBrowserName() === 'chrome') {
+                        atb = ATB.getChromeCounteractExpATB(settings.getSetting(atb).atb)
```

Comment:
I'd like to confirm that `settings.getSetting(atb)` will return a value, even in the case when extension is installed directly from the Chrome store